conpot/tests/test_pysnmp_wrapper.py

```diff
@@ -32,25 +32,38 @@ def test_wrapper_processing(self):
         """
         Tests that the wrapper can process a valid mib file without errors.
         """
-        result = mib2pysnmp('conpot/tests/data/VOGON-POEM-MIB.mib')
-        self.assertTrue('mibBuilder.exportSymbols("VOGON-POEM-MIB"' in result,
-                        'mib2pysnmp did not generate the expected output. Output: {0}'.format(result))
+        tmpdir = tempfile.mkdtemp()
+        result = mib2pysnmp2('conpot/tests/data/VOGON-POEM-MIB.mib', tmpdir)
+        self.assertTrue(result and self.check_content(os.path.join(tmpdir, 'VOGON-POEM-MIB.py')),
+                        'mib2pysnmp2 did not generate the expected output.')
+
+    def check_content(self, pyfile):
+        ret = False
+        with open(pyfile) as f:
+            for l in f.readlines():
+                if 'mibBuilder.exportSymbols("VOGON-POEM-MIB"' in l:
+                    ret = True
+                    break
+
+        return ret
 
     def test_wrapper_output(self):
         """
         Tests that the wrapper generates output that can be consumed by the command responder.
         """
         tmpdir = None
+
         try:
+            result = None
             tmpdir = tempfile.mkdtemp()
-            result = mib2pysnmp('conpot/tests/data/VOGON-POEM-MIB.mib')
-
-            with open(os.path.join(tmpdir, 'VOGON-POEM-MIB' + '.py'), 'w') as output_file:
-                output_file.write(result)
 
-            cmd_responder = command_responder.CommandResponder('', 0, [tmpdir])
-            cmd_responder.snmpEngine.msgAndPduDsp.mibInstrumController.mibBuilder.loadModules('VOGON-POEM-MIB')
-            result = cmd_responder._get_mibSymbol('VOGON-POEM-MIB', 'poemNumber')
+            if mib2pysnmp2('conpot/tests/data/VOGON-POEM-MIB.mib', tmpdir):
```

Comment:
I think we should refrain from doing paths like this and always use os.path.join for better path separator coverage on various platforms.